pandas/tests/io/test_fsspec.py

```diff
@@ -223,11 +221,12 @@ def test_s3_parquet(s3_resource, s3so):
     tm.assert_equal(df1, df2)
 
 
+@td.skip_if_installed("fsspec")
```

Comment:
extra skip here

Reply:
@jreback fixed this, merged master, green except for Travis CI